source/_posts/bars-to-code-at.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Seven Bars to Code at in Brooklyn
+date: 2017-05-03
+category: Notes
+lede: In my opinion, bars are a better place to code than coffee shops.
+author: Patrick Steadman
+published: true
+image: "/images/coding-at-bar.jpg"
+---
+
+<center>![bars to code at](/images/coding-at-bar.jpg)</center>
+
+__In my opinion, bars are a better place to code than coffee shops.__
+
+First, bars are essentially hardwood standing desks. In general, the surfaces in
+bars tend to be flat and steady. I hate the feeling of trying to code at a
+wobbly little cafe table.
+
+Second, bars tend to have more power outlets. Most recently-opened bars have
+power strips running under the bar or along seating areas. I think this is
+because bars want people to linger, and coffee shops don't. Someone staying at a
+bar for a few hours will likely order a number of expensive drinks, where
+someone staying at a cafe will only order a coffee and maybe a snack.
+
+Third, bars tend to be darker and cooler, relative to cafes, which are often
+stuffy and sunny.
+
+Finally, after a good coding session, I often find myself in a sort of distant,
+aphasic state. Everything seems flat and it's hard to think of anything to say.
+Being at a bar gives me a chance to turn off my computer, find someone to talk
+to, and reintegrate.
+
+Of course, not all bars are good places to code. Some bars are problematic for
+social reasons, and others are problematic for practical reasons. Here's a list
+of bars that Computer Lab members have found suitable for computer use.
+
```

Comment:
Consider removing first bits of each paragraph (first,second,third,finally,of course)